<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="55" started-at="2016-11-17T19:21:51Z" finished-at="2016-11-17T19:21:51Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="55" started-at="2016-11-17T19:21:51Z" finished-at="2016-11-17T19:21:51Z">
      <class name="lab.listner">
        <test-method status="FAIL" signature="actionPerformed(java.awt.event.ActionEvent)[pri:0, instance:lab.listner@2d45ba]" name="actionPerformed" duration-ms="0" started-at="2016-11-17T11:21:51Z" finished-at="2016-11-17T11:21:51Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method actionPerformed requires 1 parameters but 0 were supplied in the @Test annotation.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method actionPerformed requires 1 parameters but 0 were supplied in the @Test annotation.
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:198)
	at org.testng.internal.Parameters.createParameters(Parameters.java:134)
	at org.testng.internal.Parameters.createParameters(Parameters.java:370)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:447)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- actionPerformed -->
      </class> <!-- lab.listner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
